cypress/integration/learn/redirects/breadcrumbs.js

```diff
@@ -0,0 +1,32 @@
+/* global cy */
+
+const challengeUrl =
+  '/learn/responsive-web-design/basic-html-and-html5/say-hello-to-html-elements';
+
+const superBlockUrl = '/learn/responsive-web-design';
+const courseUrl = '/learn/responsive-web-design/#basic-html-and-html5';
+
+describe('The breadcumbs should work corectly', () => {
+  it('It should have a superblock and a course', () => {
+    cy.visit(challengeUrl);
+    cy.get('.ellipsis').contains('Responsive Web Design').and('be.visible');
+    cy.get('.breadcrumb-left')
+      .should('have.attr', 'href')
+      .and('include', superBlockUrl);
+    cy.get('.breadcrumb-right')
+      .contains('Basic HTML and HTML5')
+      .and('be.visible');
+    cy.get('.breadcrumb-right')
+      .should('have.attr', 'href')
+      .and('include', courseUrl);
+  });
+
+  it('Should redirect to the right url', () => {
+    cy.visit(challengeUrl);
+    cy.get('.breadcrumb-left', { timeout: 10000 }).click();
```

Comment:
```suggestion
    cy.get('.breadcrumb-left').click();
```
Is the timeout necessary?  In principle those should be clickable immediately.

Reply:
I think it helps with the timing issue